src/searchbar.cpp

```diff
@@ -65,10 +66,14 @@ SearchBarLineEdit::SearchBarLineEdit(QWidget *parent) :
     setToolTip(gt("search"));
     m_completer.setCompletionMode(QCompleter::UnfilteredPopupCompletion);
     m_completer.setCaseSensitivity(Qt::CaseInsensitive);
-    m_completer.setMaxVisibleItems(16);
+
+    /* The items should be less than fetch size to enable scrolling. */
+    m_completer.setMaxVisibleItems(8);
```

Comment:
8 is a magic number. Why not `SuggestionListWorker::getFetchSize() - 1` (which also provides for a longer list of visible suggestions)?

Reply:
I will use `SuggestionListWorker::getFetchSize() / 2` here. Reason being endless suggestions need a scroller to work (otherwise no scrolling could happen...), which the view wouldn't have if it can contain all the entries without scrolling.

Reply:
Done

Reply:
> I will use `SuggestionListWorker::getFetchSize() / 2` here.

Why won't `SuggestionListWorker::getFetchSize() - 1` work?